corehq/apps/accounting/models.py

```diff
@@ -340,6 +346,27 @@ def get_account_by_domain(cls, domain):
             ).filter(created_by_domain=domain).latest('date_created')
         return None
 
+    @property
+    def autopay_card(self):
+        if not self.auto_pay_enabled:
+            return None
+
+        return StripePaymentMethod.objects.get(web_user=self.auto_pay_user).get_autopay_card(self)
+
+    def update_autopay_user(self, new_user):
+        from corehq.apps.accounting.tasks import send_autopay_card_removed_email, send_autopay_card_added_email
```

Comment:
maybe this function belongs in ```accounting.tasks``` - the inline imports are a sign things should be moved